main/perl-test-inter/APKBUILD

```diff
@@ -2,16 +2,14 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=perl-test-inter
 _pkgreal=Test-Inter
-pkgver=1.07
+pkgver=1.09
 pkgrel=0
 pkgdesc="framework for more readable interactive test scripts"
 url="http://search.cpan.org/dist/Test-Inter/"
 arch="noarch"
-license="GPL PerlArtistic"
-cpandepends=""
-cpanmakedepends="   "
-depends="$cpandepends"
-makedepends="perl-dev $cpanmakedepends perl-module-build"
+license="GPL-1.0-or-later OR Artistic-1.0-Perl"
+makedepends="perl-dev perl-module-build"
+checkdepends="perl-test-pod perl-test-pod-coverage perl-file-find-rule"
```

Comment:
`perl-file-find-rule` is in `community` repository.

Reply:
moved it and its dependencies to community